pkg/ccl/backupccl/create_scheduled_backup.go

```diff
@@ -837,6 +837,12 @@ func makeScheduledBackupEval(
 			// All backups are full cluster backups for free users.
 			eval.fullBackupRecurrence = eval.recurrence
 			eval.recurrence = nil
+			if schedule.FullBackup == nil {
+				p.BufferClientNotice(ctx,
+					pgnotice.Newf("Without an enterprise license,"+
+						" this schedule only contains full backups. To mute this warning, "+
+						"add the 'FULL BACKUP ALWAYS' clause to your CREATE SCHEDULE command."))
```

Comment:
nit: Without an enterprise license this schedule will only run full backups

Personally, I'd drop the mute part but don't have a strong preference. If you decide to keep it s/warning/notice/g

Reply:
I'd also just try it out to make sure the formatting isn't wonky, incase you haven't already.

Reply:
Done.  I kept the mute part 🙃